web-api/src/persistence/postgres/workitems/mapper.ts

```diff
@@ -0,0 +1,76 @@
+import { Case } from '@shared/business/entities/cases/Case';
+import { NewWorkItemKysely } from '@web-api/database-types';
+import { RawWorkItem, WorkItem } from '@shared/business/entities/WorkItem';
+import { transformNullToUndefined } from '@web-api/persistence/postgres/utils/transformNullToUndefined';
+
+function pickFields(workItem) {
+  return {
+    assigneeId: workItem.assigneeId,
+    assigneeName: workItem.assigneeName,
+    associatedJudge: workItem.associatedJudge,
+    associatedJudgeId: workItem.associatedJudgeId,
+    caseIsInProgress: workItem.caseIsInProgress,
+    completedAt: workItem.completedAt,
+    completedBy: workItem.completedBy,
+    completedByUserId: workItem.completedByUserId,
+    completedMessage: workItem.completedMessage,
+    createdAt: workItem.createdAt,
+    docketEntry: JSON.stringify(workItem.docketEntry),
+    docketNumber: workItem.docketNumber,
+    hideFromPendingMessages: workItem.hideFromPendingMessages,
+    highPriority: workItem.highPriority,
+    inProgress: workItem.inProgress,
+    isInitializeCase: workItem.isInitializeCase,
+    isRead: workItem.isRead,
+    section: workItem.section,
+    sentBy: workItem.sentBy,
+    sentBySection: workItem.sentBySection,
+    sentByUserId: workItem.sentByUserId,
+    updatedAt: workItem.updatedAt,
+    workItemId: workItem.workItemId,
+  };
+}
+
+function getWorkItemSection({
+  section,
+  documentTitle,
+}: {
+  section: string;
+  documentTitle: string;
+}) {
+  // We have sections for caseServicesSupervisor and clerkofcourt, but as far as we can tell, they aren't used.
+  // Instead, we need to translate these into either the petitions section or the docket section depending
+  // on the document type.
+  if (!['caseServicesSupervisor', 'clerkofcourt'].includes(section)) {
+    return section;
+  }
+  if (documentTitle.toLocaleLowerCase() == 'petition') {
+    return 'petitions';
+  }
+  return 'docket';
+}
```

Comment:
This is not the ideal way to handle this. However, we are trying to avoid wide-sweeping changes to how it was previously working to avoid breaking anything.

Reply:
Before, the duplicate work items (in section-outbox) were assigned to, e.g., docket or petitions, but the "main" work item could be assigned to `caseServicesSupervisor` or `clerkofthecourt`. This section data was otherwise unused, and filtering for QC items relied on outbox item sections being `docket` or `petitions`, so the main work item now just uses `docket` or `petitions` regardless of who creates the work item. (And we can still see who created the work item--e.g., a case services supervisor vs. a petitions clerk--so we aren't losing any information in the DB.)